docs/development/design/network-router.md

```diff
@@ -0,0 +1,114 @@
+# Network Router
+
+> Additional details and running code in [PR #981](https://github.com/nexodus-io/nexodus/pull/981)
+
+## Summary
+
+Add the ability for device's running nexd to access non-nexd devices and act as a router for external networks.
+
+## Proposal
+
+There are two supported use cases.
+
+1. [Network Router with SNAT](#network-router-with-snat) Traffic from the Nexd mesh can reach external non-nexd nodes via source NAT without any configuration required.
+2. [Network Router with SNAT disabled](#network-router-with-snat-disabled) Traffic from the Nexd mesh can reach external non-nexd nodes but require a route to be added for return traffic since SNAT is disabled.
+3. With SNAT disabled, a user can initiate traffic from external, non-nexd devices to initiate traffic in to the nexodus mesh.
```

Comment:
The section begins with saying there are 2 supported use cases, so I wouldn't make this a number 3 on the list. You could add it as another sentence with number 2, or just as a clarifying sentence afterwards like this.

```suggestion

With SNAT disabled, a user can initiate traffic from external, non-nexd devices to initiate traffic in to the nexodus mesh.
```